### DISCUSSION TITLE
Security issue mail or scam?

### DISCUSSION THREAD

**Discussion by 3xp11i** [balta2ar/brotab#90] [General]
Hey @balta2ar, can you please tell me what is this mail for? And are you aware of it? or is it something like a scam?

![image](https://user-images.githubusercontent.com/65062036/215662769-73989cea-7db4-4986-b7af-e7adcb7392e1.png)

**Comment by balta2ar** [comment]
@prakhartiwari0 No idea what this is, but looks like a nicely crafted spam message, if you ask me 😅

**Reply by 3xp11i** [reply]
LOL, I thought the same that's why asked you about it. 
But whoever is doing this is pretty smart, I mean who could try to scam *developers* through a platform like **GitHub**, so is the level of their creativity!
